sql/src/main/java/org/apache/druid/sql/calcite/schema/DruidSchema.java

```diff
@@ -791,7 +795,7 @@ DruidTable buildDruidTable(final String dataSource)
     } else {
       tableDataSource = new TableDataSource(dataSource);
     }
-    return new DruidTable(tableDataSource, builder.build(), null, isJoinable, isBroadcast);
+    return new DruidTable(tableDataSource, ROW_SIGNATURE_INTERNER.intern(builder.build()), null, isJoinable, isBroadcast);
```

Comment:
Since there's no advantage or disadvantage  of updating it, should I change it to maintain consistency in the class or revert it since it provides no benefit.

Reply:
I will suggest reverting it since it is simpler that way.

Reply:
👍 Updated